apis/metal3.io/v1alpha1/baremetalhost_validation_test.go

```diff
@@ -71,6 +71,17 @@ func TestValidateCreate(t *testing.T) {
 			oldBMH:    nil,
 			wantedErr: "hardwareRAIDVolumes and softwareRAIDVolumes can not be set at the same time",
 		},
+		{
```

Comment:
I think, we can add more test like happy path or more edge cases.

Reply:
Okay, I will add a few more test cases.